src/Components/Nav/Nav.tsx

```diff
@@ -4,20 +4,21 @@ import NavLink from "../NavLink/NavLink";
 import { default as NavLinkStyles } from "./Nav.module.scss";
 import bemCssModule from "bem-css-modules";
 import { useTranslation } from "react-i18next";
-import { Link } from "constants/types/Link";
+import { LinkElements } from "constants/types/Link";
+
 const style = bemCssModule(NavLinkStyles);
 
 const Nav = () => {
   const { t } = useTranslation();
 
-  const links: Link[] = [
+  const links: LinkElements[] = [
     {
       title: `${t("menu.pos1")}`,
-      to: "/task",
+      to: "/",
     },
     {
       title: `${t("menu.pos2")}`,
```

Comment:
Please use other naming convention. The naming like pos1, pos2 is not clear and if you want to change order of items there will be a lot of changes.